packages/web-components/fast-foundation/src/utilities/typings.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Helper for enumerating a type from a const object
+ * Example: export type Foo = ValuesOf<typeof Foo>
+ */
+export type ValuesOf<T> = T[keyof T];
```

Comment:
Is this meant to be internal to Foundation only or exported? Should it be marked as internal or public for api-extractor?

Reply:
This should be marked as public. While it could be internal, I think if folks want to use this the export for public use is key for not needing to recreate the type.

Reply:
Other than that, lgtm